src/openrct2/config/Config.cpp

```diff
@@ -99,6 +99,14 @@ namespace Config
         ConfigEnumEntry<sint32>("CELSIUS", TEMPERATURE_FORMAT_C),
         ConfigEnumEntry<sint32>("FAHRENHEIT", TEMPERATURE_FORMAT_F),
     });
+    
+    static auto Enum_ScaleQuality = ConfigEnum<sint32>(
+    {
+        ConfigEnumEntry<sint32>("NEAREST_NEIGHBOR", SCALE_QUALITY_NN),
```

Comment:
You missed a `U`.

Reply:
(The code base uses British spelling.)

Reply:
Whoops, too accustomed to US English I guess.

Reply:
This one is still missing a `U`.

Reply:
And apparently one in the changelog too, fixed it.